scripts/release-workaround.py

```diff
@@ -0,0 +1,280 @@
+                            "sdk-android-release.aar")
+    elif module == Module.CRYPTO:
+        return os.path.join(root_project_dir, "crypto/crypto-android/build/outputs/aar",
+                            "crypto-android-release.aar")
+    else:
+        raise ValueError(f"Unknown module: {module}")
+
+
+def get_publish_task(module: Module) -> str:
+    if module == Module.SDK:
+        return ":sdk:sdk-android:publishToSonatype"
+    elif module == Module.CRYPTO:
+        return ":crypto:crypto-android:publishToSonatype"
+    else:
+        raise ValueError(f"Unknown module: {module}")
+
+
+def run_publish_close_and_release_tasks(root_project_dir, publish_task: str):
+    gradle_command = f"./gradlew {publish_task} closeAndReleaseStagingRepository"
+    result = subprocess.run(gradle_command, shell=True, cwd=root_project_dir, text=True)
+    if result.returncode != 0:
+        raise Exception(f"Gradle tasks failed with return code {result.returncode}")
+
+
+github_token = os.environ['GITHUB_API_TOKEN']
+if github_token is None:
+    print("Please set GITHUB_API_TOKEN environment variable")
+    exit(1)
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-m", "--module", type=module_type, required=True,
+                    help="Choose a module (SDK or CRYPTO)")
+parser.add_argument("-v", "--version", type=str, required=True,
+                    help="Version as a string (e.g. '1.0.0')")
+parser.add_argument("-r", "--ref", type=str, required=True,
+                    help="Ref to the git matrix-rust-sdk (branch name, commit or tag)")
+
+args = parser.parse_args()
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+project_root = os.path.dirname(current_dir).rstrip(os.sep)
+sdk_git_url = "https://github.com/matrix-org/matrix-rust-sdk.git"
+
+print(f"Project Root Directory: {project_root}")
+print(f"Selected module: {args.module}")
+print(f"Version: {args.version}")
+print(f"SDK git ref: {args.ref}")
+
+build_version_file_path = get_build_version_file_path(args.module, project_root)
+major, minor, patch = read_version_numbers_from_kotlin_file(build_version_file_path)
+if is_provided_version_higher(major, minor, patch, args.version):
+    print(
+        f"The provided version ({args.version}) is higher than the previous version ({major}.{minor}.{patch}) so we can start the release process")
+else:
+    print(
+        f"The provided version ({args.version}) is not higher than the previous version ({major}.{minor}.{patch}) so bump the version before retrying.")
+    exit(0)
+
+with TemporaryDirectory() as sdk_path:
+    clone_repo_and_checkout_ref(sdk_path, sdk_git_url, args.ref)
+    linkable_ref = get_linkable_ref(sdk_path, args.ref)
+    # Call the build script in debug mode first to workaround a bug in cargo-ndk
+    execute_build_script(current_dir, sdk_path, args.module, False)
+    # Then use the proper release mode one
+    execute_build_script(current_dir, sdk_path, args.module, True)
```

Comment:
The release script clones the SDK repo, so I had to add the 'build debug, then release' workaround here and duplicate the script.